draft-ietf-jsonpath-base.md

```diff
@@ -311,11 +323,15 @@ or the more compact *dot notation*, for example:
 $.store.book[0].title
 ~~~~
 
-to build paths that are input to a JSONPath implementation.
-A single path may use a combination of bracket and dot notations.
+A JSONPath expression may use a combination of bracket and dot notations.
 
 Dot notation is merely a shorthand way of writing certain bracket notations.
 
+This document treats the bracket notations as canonical and defines various shorthands in terms
```

Comment:
Do you mean "as canonical but uses the bracket shorthand in examples and descriptions"?

Reply:
Yes, this could be expanded.  The idea is that we define JSONPath in terms of bracketed steps, but as syntactic sugar we also define a few shorthands.

Reply:
I added a sentence along the lines @timbray suggested.